app/models/journable/with_historic_attributes/loader.rb

```diff
@@ -86,6 +116,28 @@ def journalized_class
       journables.first.class
     end
 
+    def load_journals_by_id(journal_ids)
+      Journal
+        .where(id: journal_ids)
+        .includes({ customizable_journals: :custom_field })
+        .index_by(&:id)
```

Comment:
Tiny thing: The `Journal` needn't be involved in the loading. The same result can be achieved by

```
Journal::CustomizableJournal
  .where(journal_id: journal_ids)
  .includes(:custom_field)
  .index_by(&:journal_id)
```

Reply:
Done 57c3bf093da05728519db5c0e28f267524258281